fastplotlib/graphics/selectors/_linear.py

```diff
@@ -355,6 +365,6 @@ def _move_graphic(self, delta: np.ndarray):
         """
 
         if self.axis == "x":
-            self.selection = self.selection() + delta[0]
+            self._selection = self._selection() + delta[0]
         else:
-            self.selection = self.selection() + delta[1]
+            self._selection = self._selection() + delta[1]
```

Comment:
once `_selection` is defined in the construction just access it using the property.